pkg/operation/seed/types.go

```diff
@@ -45,7 +45,5 @@ type Components struct {
 
 // DNS contains all necessary DNS components for the Seed cluster.
 type DNS struct {
```

Comment:
We don't need this struct anymore now, or?

Reply:
The struct still has one field `record` used to create the managed Ingress `DNSRecord`.
See https://github.com/gardener/gardener/blob/master/pkg/operation/seed/seed.go#L1615

Reply:
done